Dockerfile

```diff
@@ -4,10 +4,17 @@ COPY . .
 RUN make build
 
 FROM quay.io/centos/centos:stream9
-LABEL maintainers="The KubeVirt Project <kubevirt-dev@googlegroups.com>"
-LABEL description="KubeVirt CSI Driver"
+ARG git_url=https://github.com/kubevirt/cdi-driver.git
 
-RUN dnf install -y e2fsprogs xfsprogs && dnf clean all
-COPY --from=builder /src/kubevirt-csi-driver/kubevirt-csi-driver .
+LABEL maintainers="The KubeVirt Project <kubevirt-dev@googlegroups.com>" \
+      description="KubeVirt CSI Driver" \
+      multi.GIT_URL=${git_url}
 
 ENTRYPOINT ["./kubevirt-csi-driver"]
```

Comment:
Why the entrypoint is before even copying the file, I am not sure if that is the official style but I often see ENTRYPOINT as last entry

Reply:
I modified the order to optimize docker-build caching: anything that is more likely to be changed, should be after things that are more constant.

There is no good reason why the entry point is at the end - it will work the same.